README_BUILD.md

```diff
@@ -24,12 +24,7 @@ Install to build python backends
 ```shell
 analog@precision:~$ sudo apt-get install python3 python3-pip python3-setuptools
 ```
-Install to Read local context attributes from `/etc/libiio.ini`
-```shell
-analog@precision:~$ git clone https://github.com/pcercuei/libini.git
-analog@precision:~$ cd libini
-analog@precision:~/libini$ mkdir build && cd build && cmake ../ && make && sudo make install
-```
+
```

Comment:
Why remove this?

Reply:
libini is now included in the libiio repo, and this step isn't required anymore.

https://github.com/analogdevicesinc/libiio/tree/master/libini

Reply:
Alright.